data-prepper-plugins/elasticsearch/README.md

```diff
@@ -42,13 +42,36 @@ pipeline:
 
 The elasticsearch sink will reserve `otel-v1-apm-service-map` as index for record ingestion.
 
+### Amazon Elasticsearch Service (AES)
```

Comment:
nit - would replace `AES` with `Amazon ES` throughout

Reply:
Apparently we have legal restrictions on the name - please use Amazon Elasticsearch Service; not AES or Amazon ES even

Reply:
@yadavcbala Thanks for call out!

Reply:
@yadavcbala: Oh interesting, do you know where that's coming from? We recently published the blog post with legal's approval, the guidance given was to use Amazon Elasticsearch Service first, then optionally Amazon ES through the rest of the document. We can sync on this offline

Reply:
If the guidance you got is from Legal - I think you are right, The one i got is from a forwarded email and it was long back. There were couple of approved names given in that but the gist was to prefer Amazon Elasticsearch Service. I took it as use Amazon Elasticsearch Service :-)

Reply:
Cool, thanks for the info. Using the full name is never wrong so we're good in this PR 👍